polkadot/zombienet_tests/elastic_scaling/0001-basic-3cores-6s-blocks.toml

```
@@ -0,0 +1,42 @@
+[settings]
+timeout = 1000
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.async_backing_params]
+  max_candidate_depth = 6
+  allowed_ancestry_len = 2
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.scheduler_params]
+  max_validators_per_core = 1
+  scheduling_lookahead = 2
+  num_cores = 3
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.approval_voting_params]
+  needed_approvals = 3
+  max_approval_coalesce_count = 5
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+default_command = "polkadot"
+
+  [relaychain.default_resources]
+  limits = { memory = "4G", cpu = "2" }
+  requests = { memory = "2G", cpu = "1" }
+
+  [[relaychain.node_groups]]
+  name = "elastic-validator"
+  count = 5
+  args = [ "-lparachain=debug,parachain::candidate-backing=trace,parachain::provisioner=trace,parachain::prospective-parachains=trace,runtime=debug"]
+
+{% for id in range(2000,2002) %}
+[[parachains]]
+id = {{id}}
+addToGenesis = true
+
+    [parachains.collator]
+    name = "some-parachain"
+    image = "{{CUMULUS_IMAGE}}"
```

```suggestion
    image = "{{COL_IMAGE}}"
```
This should be `COL_IMAGE` to use adder-collator.